src/js/core/widget/core/Appbar.js

```diff
@@ -397,6 +404,34 @@
 				}
 			};
 
+			/**
+			 * Toggle visibility of BottomBar
+			 * @method _toggleSelectAll
+			 * @param {boolean} visible
+			 * @member ns.widget.core.Appbar
+			 * @protected
+			 */
+			prototype._toggleBottomBar = function (visible) {
+				var bottomBar = this._ui.bottomBar;
+
+				if (visible) {
```

Comment:
What do you think about using `classList.toggle(classes.hidden, true)` instead of this condition expression?

Reply:
Actually, I can use `toggle`. I usually wonder if the system supports this, but I checked that we use this in several places. I'll change this.